src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/NameScope.cs

```diff
@@ -18,8 +18,6 @@ namespace System.Xaml
     /// </summary>
     internal class NameScope : INameScopeDictionary
     {        
-        #region INameScope
```

Comment:
I think we should keep the #region statements. Open to comments, if you think otherwise

Reply:
I think the .NET style guide no longer favours regions.